src/components/shared/formItems/ProjectDiscord.tsx

```diff
@@ -11,7 +11,7 @@ export default function ProjectDiscord({
     <Form.Item
       name={name}
       label={hideLabel ? undefined : 'Discord'}
-      extra="An invite link to your project's Discord server."
```

Comment:
**question**: what's the reason for removing this?

Reply:
@holenyc did you mean to resolve this comment without addressing it?